app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,48 @@
+<h2 class="d-flex center">Sign up</h2>
+
+<div class="middle m-33 p-2 wrap">
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, required: true, autocomplete: "email", class:"form-title" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :name %><br />
+      <%= f.text_field :name, autofocus: true, required: true, autocomplete: "name", class:"form-title" %>
+    </div>
```

Comment:
- [ ] Please I get this NameError error saying you have an undefined local variable or method `confirmed_at' for the User model when signing up so make sure you fix this and one can sign up successfully. :pray: 
  
  ![image](https://github.com/emarkees/Gist_Blog/assets/92303891/ffa6910a-b2fc-4434-8ac3-654424717c6b)

Reply:
done